[2025-01-04T04:16:56.437+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-04T04:16:56.454+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: call_9851100067_2025-01-04T04-15-31-903380Z.find_if_proper_interaction_is_completed scheduled__2025-01-04T04:15:31.903380+00:00 [queued]>
[2025-01-04T04:16:56.459+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: call_9851100067_2025-01-04T04-15-31-903380Z.find_if_proper_interaction_is_completed scheduled__2025-01-04T04:15:31.903380+00:00 [queued]>
[2025-01-04T04:16:56.459+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-04T04:16:56.465+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): find_if_proper_interaction_is_completed> on 2025-01-04 04:15:31.903380+00:00
[2025-01-04T04:16:56.470+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=173335) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-04T04:16:56.471+0000] {standard_task_runner.py:72} INFO - Started process 173336 to run task
[2025-01-04T04:16:56.471+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'call_9851100067_2025-01-04T04-15-31-903380Z', 'find_if_proper_interaction_is_completed', 'scheduled__2025-01-04T04:15:31.903380+00:00', '--job-id', '981', '--raw', '--subdir', 'DAGS_FOLDER/complete_dag.py', '--cfg-path', '/tmp/tmpysuj1pox']
[2025-01-04T04:16:56.471+0000] {standard_task_runner.py:105} INFO - Job 981: Subtask find_if_proper_interaction_is_completed
[2025-01-04T04:16:56.494+0000] {task_command.py:467} INFO - Running <TaskInstance: call_9851100067_2025-01-04T04-15-31-903380Z.find_if_proper_interaction_is_completed scheduled__2025-01-04T04:15:31.903380+00:00 [running]> on host 680cb867086b
[2025-01-04T04:16:56.534+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='call_9851100067_2025-01-04T04-15-31-903380Z' AIRFLOW_CTX_TASK_ID='find_if_proper_interaction_is_completed' AIRFLOW_CTX_EXECUTION_DATE='2025-01-04T04:15:31.903380+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-04T04:15:31.903380+00:00'
[2025-01-04T04:16:56.534+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-04T04:16:56.539+0000] {base.py:84} INFO - Retrieving connection 'aicallagent'
[2025-01-04T04:16:56.543+0000] {sql.py:544} INFO - Running statement: UPDATE user_details SET status = 'answered', modified_on_utc = now() at time zone 'utc' WHERE caller_id = 'demo-scheduled-from-ui-global271ea0e8-022f-4ef5-bedd-53bbec7b31a2', parameters: None
[2025-01-04T04:16:56.547+0000] {sql.py:553} INFO - Rows affected: 1
[2025-01-04T04:16:56.548+0000] {logging_mixin.py:190} INFO - Marked record demo-scheduled-from-ui-global271ea0e8-022f-4ef5-bedd-53bbec7b31a2 as answered.
[2025-01-04T04:16:56.548+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-04T04:16:56.552+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-04T04:16:56.552+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=call_9851100067_2025-01-04T04-15-31-903380Z, task_id=find_if_proper_interaction_is_completed, run_id=scheduled__2025-01-04T04:15:31.903380+00:00, execution_date=20250104T041531, start_date=20250104T041656, end_date=20250104T041656
[2025-01-04T04:16:56.568+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-04T04:16:56.577+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-04T04:16:56.578+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
